test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -5078,6 +5078,104 @@ BOOST_AUTO_TEST_CASE(invalid_address_length)
 	CHECK_WARNING(text, "checksum");
 }
 
+BOOST_AUTO_TEST_CASE(interface)
+{
+	char const* text = R"(
+		interface I {
+		}
+	)";
+	success(text);
+}
+
+BOOST_AUTO_TEST_CASE(interface_constructor)
+{
+	char const* text = R"(
+		interface I {
+			function I();
+		}
+	)";
+	success(text);
```

Comment:
This doesn't fail for me on 0.4.10:

```
contract A {
    function A();
}
```

What is your sample code which has internal compiler error?